doc/source/template_nocontainer_nextflow.config

```diff
@@ -40,8 +46,3 @@ process {
 
 // OTHER SETTINGS:
 // env.SOME_ENV_VARIABLE = ''
-
-// params {
-//     nThreadsParalogs = 
-//     nThreadsOrthologs = 
-// }
```

Comment:
As far as I can see this template still has a `container` setting.

Reply:
Not needed indeed, thanks.